components/filter/events.tsx

```diff
@@ -214,6 +214,15 @@ interface EventProps {
 }
 
 const Event = ({ visible, event, jsonLd }: EventProps) => {
+  const [thumbnail, setFallbackImage] = useState(event.thumbnail);
+  const handleImageError = () => {
+    const tinaUrl = /https:\/\/assets\.tina\.io\/[^/]+\/(.*)/;
+    const match = event.thumbnail.match(tinaUrl);
+    if (match) {
+      setFallbackImage(`/images/${match[1]}`);
+    }
+  };
```

Comment:
given that the thumbnail urls have been migrated... do we need this?

I'd fallback to a static image instead of rewriting the url

Reply:
The point to files in the repo but not all of the files have been synced to Tina cloud. I've had issues syncing the repo with Tina cloud because of the size of the repo.